src/clj/rems/api/forms.clj

```diff
@@ -88,15 +90,22 @@
       (ok (form/get-form-template form-id)))
 
     (POST "/create" []
-      :summary "Create form"
+      :summary "Create or edit form"
```

Comment:
Why not have two separate endpoints? Create is when a completely new form is created and update is when one is, well, updated?

Reply:
You're right (as well as Esko who commented about the same): better to have the API as clean as possible. Done.